scoring.js

```diff
@@ -0,0 +1,70 @@
+function calculateScore(player) {
+  switch (player.position) {
+    case 'QB':
+      return passingScore(player) + rushingScore(player)
+
+    case 'RB':
+      return (
+        rushingScore(player) +
+        receivingScore(player) +
+        puntreturnScore(player) +
+        kickreturnScore(player)
+      )
+
+    case 'WR':
+      return (
+        receivingScore(player) +
+        rushingScore(player) +
+        kickreturnScore(player) +
+        puntreturnScore(player)
+      )
```

Comment:
_Nit_ Wide Receiver and Running Back are calculated the same way, repetitive code.